api/client/node/inspect.go

```diff
@@ -26,10 +25,13 @@ func newInspectCommand(dockerCli *client.DockerCli) *cobra.Command {
 	var opts inspectOptions
 
 	cmd := &cobra.Command{
-		Use:   "inspect [OPTIONS] self|NODE [NODE...]",
-		Short: "Display detailed information on one or more nodes",
-		Args:  cli.RequiresMinArgs(1),
+		Use:   "inspect [OPTIONS] [NODE...]",
+		Short: "Display detailed information on one or more nodes (default to current node)",
+		Args:  nil,
```

Comment:
Instead of `nil` could you use `cobra.ArbitraryArgs` so it's more explicit about what args are accepted?